src/app/component/user/components/profile/profile-service/profile.service.ts

```diff
@@ -29,7 +29,9 @@ export class ProfileService {
   }
 
   public getFactsOfTheDay(): Observable<CardModel> {
-    return this.http.get<CardModel>(`${this.backEnd}facts/dayFact/2`);
+    const currentLang = this.languageService.getCurrentLanguage();
```

Comment:
Why do you need a language there?

Reply:
Because there are translations for all facts of the day in database, and we show the user the fact in the language which he chooses